core/graph/builders/build_graph_batched.py

```
@@ -0,0 +1,204 @@
+                        edges.append({
+                            '_from': f"arxiv_papers/{papers[i]['_key']}",
+                            '_to': f"arxiv_papers/{papers[j]['_key']}",
+                            'weight': 0.95,
+                            'month': month,
+                            'category': category
+                        })
+                        
+                        if len(edges) >= 10000:
+                            # Insert batch
+                            self.temporal_edges.insert_many(edges)
+                            edge_count = len(edges)
```

Ruff (0.12.2)

Local variable `edge_count` is assigned to but never used

Remove assignment to unused variable `edge_count`

(F841)